chain/chaintest/balance_handler_test_helpers.go

```diff
@@ -0,0 +1,56 @@
+// Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package chaintest
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/ava-labs/hypersdk/chain"
+	"github.com/ava-labs/hypersdk/codec/codectest"
+	"github.com/ava-labs/hypersdk/state/tstate"
+)
+
+// TestBalanceHandler tests [b] by doing the following:
```

Comment:
The pattern of `[name]` is something we do in our codebase but I don't think is standard.... in fact it actually breaks indexing in my ide (e.g if I have a struct named `Foo`, calling it `Foo` in a comment indexes it in the IDE, but `[Foo]` will not).

Reply:
Done